### ISSUE TITLE
Make it easier to recognize hold / exhale / inhale

### ISSUE THREAD

**Issue by mahnouel** [filipeisho/breathe#3]
Maybe the color of the circle could change based on what the user should do? This way it would be easier to write code / be with your eyes on another part of the screen and still "see" at which stage you are. 👀

**Comment by tryhardfifi** [comment]
You are totally right! It is very difficult to know when a step starts/finishes. Initially I thought colors wouldn't help because of [this](https://biology.stackexchange.com/questions/24506/can-the-human-eye-distinguish-colors-in-the-periphery) (someone posted it on the HN thread),  but yesterday I wrote a quick demo and saw the colors peripherally perfectly (probably the corner of the screen is not so far away). 

I have almost finished all the other features, as soon as I do I will release it 🚀

**Comment by sojusnik** [comment]
An option to adjust the colors of all periods (inhale, exhale, hold) would help too.

The transition time between those colors should be fluent (about 1 second) and not abrupt, because this would draw too much attention, while a period should end with a transition, not start with it.

**Comment by tryhardfifi** [comment]
@sojusnik Sorry I took so long to answer, I was fixing a lot of bugs. You are totally right about this, but at the moment I am not sure how to implement this... While I think of a way I will give priority to the features I was expecting to finish for the release and we can revisit this later! (Also, if you happen to know how I could implement it, I would be glad to hear)  😄

**Comment by tryhardfifi** [comment]
Hi! The new release (that has the initial feature request) is already available! You can download it [here](https://github.com/filipeisho/breathe/releases)! I am not going to close this issue as I haven't finished implement the fluid color change between steps 😄

**Comment by themightyoarfish** [comment]
When I adjust the colors, the changes do not seem to persist. They reset to the defaults as soon as i click "Open window" (10.14.6).

**Comment by tryhardfifi** [comment]
Sorry I took so long to answer... I will look into this! Are you using breathe 2.0? It's really weird because it works just fine for me

**Comment by themightyoarfish** [comment]
Yes, I am using the latest release.

**Comment by tryhardfifi** [comment]
@themightyoarfish sorry to keep you waiting, I just found the bug. I will make a release later today or tomorrow. ~~To fix the bug I needed to change the 'Done' button to a 'Change' button, so after choosing the colors you need to press Change or else they won't be saved.~~ Edit: I found a way to do it without changing the button. The release is up, you can download it [here](https://github.com/filipeisho/breathe/releases). Thank you a lot for pointing out the bug!

**Comment by tryhardfifi** [comment]
Hi @themightyoarfish there was another bug that prevented the color panel from showing up, I just released version 2.0.2 and now everything should be working as expected. 😄

**Comment by themightyoarfish** [comment]
Cool thanks! The color changing works now. I have this new issue that when breathe window is visible and I click the icon in  the menu bar, the options open, but the breathe window disappears. Not a  huge deal; i can adjust the  colors or options without seeing the window, but kinda surprising.

**Comment by tryhardfifi** [comment]
That is expected behaviour. The feature of breathe's window being always on top and being able to transport it between spaces was implemeted using NSAlert, which means that before dismissing breathe's window you aren't able to press any other menu/window of breathe. I close the window and re-open it everytime because if not we wouldn't be able to press anything inside the settings preferences.
